packages/funding-rate-proposer/src/proposer.js

```diff
@@ -126,9 +126,13 @@ class FundingRateProposer {
     // TODO: Should allow user to set an ROI-based metric to customize whether
     // bot strategically waits to submit funding rate proposals. Rewards
     // increase with time since last update.
-    await Promise.map(Object.keys(this.contractCache), contractAddress => {
-      return this._updateFundingRate(contractAddress, usePriceFeedTime);
-    });
+    // TODO: Note we update funding rates sequentially so that we can hardcode the nonce for each proposal passing
+    // transactions to the `ynatm` package. If these calls were submitted in parallel then we wouldn't be able to
+    // hardcode the nonce, which could cause unintended reverts due to duplicate transactions. Once the `ynatm` package
+    // can handle nonce management, then we should update this logic to run in parallel.
+    for (let contractAddress of Object.keys(this.contractCache)) {
```

Comment:
can this be `const`?

Reply:
![image](https://user-images.githubusercontent.com/9457025/114433712-1ed7ff00-9b90-11eb-9b8c-5157049a0834.png)